netmiko/mikrotik/mikrotik_ssh.py

```diff
@@ -0,0 +1,115 @@
+from netmiko.cisco_base_connection import CiscoSSHConnection
+
+
+class MikrotikBase(CiscoSSHConnection):
+    """Common Methods for Mikrotik RouterOS and SwitchOS"""
+
+    def __init__(self, **kwargs):
+        if kwargs.get("default_enter") is None:
+            kwargs["default_enter"] = "\r\n"
+
+        self.in_config_mode = True
```

Comment:
Should default to False. Should be private variable i.e. `self._in_config_mode`.

Reply:
Agreed, and in testing works as expected